addons/web/static/src/less/modal.less

```diff
@@ -73,6 +73,75 @@
             }
         }
     }
+
+    @media (max-width: @screen-xs-max) {
```

Comment:
This is a lot of less and I'm not sure how much of it is really needed. When we are done working on the specifics of the actual code, I'll look into it with my dear colleague @qsm-odoo.

Reply:
okay